src/client/contract-client.js

```diff
@@ -122,7 +115,8 @@ class Client {
         // Build request.
         const action = {
           channelID: this.channelID,
-          nonce: nonceResult.nonce,
+          nonce: '3',
+          blockExpirationNumber: '3',
```

Comment:
This doesn't seem right.  I understand setting these values in the tests, but by hardcoding the block expiration number and nonce here this would not work with an actual node.  Nonce should probably be randomized.

Block Expiration Number should be set based on what the current block number is.  Will likely need to add a function to the node client to get this.

Reply:
hi @jeff1010322, is this function something we can work on on our end, or it's something only you/Ethan/Chase can do?

Reply:
Hi @jeff1010322,
in the design of blockExpirationNumber, is there a way to opt out from this "expiration" mechanism?
I mean: what if someone wants to submit a transaction with no expiration deadline? 
I was thinking maybe a negative blockExpirationNumber might encode the "Good untill cancelled" or something like that.

Reply:
Hi, I don't think we want to allow a no expiration transaction.  There is also not a way for a client to cancel a transaction once submitted.  This is something that needs to be configured in the contract client to set a value that is a certain number of blocks ahead of the current block height of the ledger.  I do think this will depend on an update to the Mazzaroth node to be able to get the current ledger height, so this change may need to wait until that is updated.